pwndbg/aglib/kernel/paging.py

```diff
@@ -197,13 +196,14 @@ def get_vmalloc_vmemmap_bases(self):
     @property
     @pwndbg.lib.cache.cache_until("stop")
     def physmap(self):
-        pob = pwndbg.aglib.symbol.lookup_symbol_addr("page_offset_base")
-        result = None
-        if pob is not None:
-            if pwndbg.aglib.memory.peek(pob):
-                result = pwndbg.aglib.memory.u64(pob)
+        result = pwndbg.aglib.kernel.symbol.try_usymbol("page_offset_base")
         if result is None:
-            return guess_physmap()
+            result = INVALID_ADDR
+            min = 0xFFFF888000000000 if self.paging_level == 4 else 0xFF11000000000000
```

Comment:
can you add simple comment, for where we can get `0xFFFF888000000000` and `0xFF11000000000000`?